recipes/vulkan-validationlayers/all/conanfile.py

```diff
@@ -57,18 +61,21 @@ def _get_compatible_spirv_tools_version(self):
         }.get(str(self.version), False)
 
     def requirements(self):
-        self.requires("spirv-tools/{}".format(self._get_compatible_spirv_tools_version), private=True)
+        #TODO set private=False, once the issue is resolved https://github.com/conan-io/conan/issues/9390
+        self.requires("spirv-tools/{}".format(self._get_compatible_spirv_tools_version), private=not self._is_cross_building())
```

Comment:
you mean set private=True once the issue is resolved.

But why not remove private for the moment?

Reply:
I have diligent-core recipe in my repository, it uses a slightly different spirv-tools. It also overrides the namespace.
If we have it not private, it breaks that recipe.